src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs

```diff
@@ -8,7 +8,7 @@
 using Microsoft.Azure.Templates.Analyzer.Cli;
 using Microsoft.Azure.Templates.Analyzer.Types;
 using Newtonsoft.Json.Linq;
-
+using System.Text.RegularExpressions;
```

Comment:
Can you put this under System.IO? We add references in alphabetical order, except for System ones that go on top

Reply:
done in [`5718325` (#297)](https://github.com/Azure/template-analyzer/pull/297/commits/5718325559dc7b595aa050102a66b33684e1dc30)